m4_sessions/1810-inning/big_data.md

```diff
@@ -1,31 +1,30 @@
 ## Group Members:
-
-
+JP Lynch
+Kurt Miller
+Gray Smith
+Joel O'Connor
 
 ## Name of Session: 
+Big Data: How Big is Big Enough?
 
 ## Session Description (Why should students come to this session): (EX: Are you interested in biological enhancement? Drop in to learn about biohacking.)
+Step into big data: what it is, how it's used, and what are the ethical ramifications.
 
 ## Session Outcomes 
-
-(What will students learn/leave with?): 
-
-
+Whta is big data?
+What tools are used in big data?
+WHat are the ethical ramifications of bigdata?
 
      Best practices:  No section should be longer than 20 minutes
      Create a session that YOU would want to attend
      Share resources in your slack channel
      All sessions NEED to have a closing activity, reflection, discussion
 ## Session Outline 
-
-(Please include who is facilitating and how much time you are going to spend in each section):
-
-## Recommended Trajectory 
-
-    ## Introductons 
-    ## Intro to new material
-    ## Small Group Activity
-    ## Closing Discussion
-
-
-Submit a PR at the end of this session and tag @emhickmann
+Student intros- name, mod, a hobby outside of coding (5-10min) All
+Presentation of what is big data- (20min) Kurt and Joel
+Presentation of what tools are used in big data- (20min) Gray
+Presentation on ethics of big data- (15min) JP
+Seperate students into 4 groups (2 for pros of big data, 2 for negatives of big data), dicuss amongst groups, each group will present in the front of the class (15min) JP
+Wrap up- (5min) All
```

Comment:
Get all of these resources in place and this will be good...especially having groups present.  I would push to think about ways that they can use these ideas while at Turing and/or in the future!